gradle-scripts/maven-publish.gradle

```diff
@@ -53,3 +50,17 @@ publishing {
         }
     }
 }
+
+signing {
+    def signingKey = System.getenv("PGP_KEY")
```

Comment:
Can we recreate those when we lose the credentials? And what is the best way to share it in team ?

Reply:
Yes, the GPG-Key can easily be recreated using GPG and uploaded to github.

Reply:
I think if we lost the key, we can just create a new one...-